### DISCUSSION TITLE
Rename dotnet formula to dotnet-sdk

### DISCUSSION THREAD

**Discussion by cglong** [Homebrew/discussions#1797] [Everyday usage] [answered]
I believe `homebrew/core/dotnet` is misnamed and should be renamed to `homebrew/core/dotnet-sdk`. The .NET SDK is similar in theory to a JDK, in that it allows a user to both build and run executables. Looking at [the `test` block](https://github.com/Homebrew/homebrew-core/blob/8a4aa29930a84d1e9c57e8d866caa11715aaeb6f/Formula/dotnet.rb#L48) of the formula, it's obvious this is the SDK.

This also seems to be the branding that's encouraged by the project. Looking at [the landing page](https://dotnet.microsoft.com/download), users are encouraged to "Download .NET SDK". In addition, the official APT package repository [names the package as `dotnet-sdk-5.0`](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#install-the-sdk).

Note that, while I'd like to just open a PR for this, I'm currently unable to do so.

**Comment by SMillerDev** [comment, marked as answer]
Since it allows users to run software too and we don't have a separate formula for the non-sdk part, I think the current naming is good enough.

**Reply by cglong** [reply]
Makes total sense! Thanks for the quick reply :)
